contributed_definitions/nyaml/NXsts.yaml

```
@@ -421,23 +413,47 @@ NXsts(NXsensor_scan):
         # parameters how to change the location from measurement to measurement during the scan
         scan_control(NXcollection):
           exists: optional
-          roi(NXtransformations):
+          scan_range(NX_NUMBER):
+            unit: NX_LENGTH
             exists: optional
+            doc: |
+              The scan range (in STM experiment) is the area that the tip will scan. (e.g.
+              5.000000E-9 5.000000E-9)
+          scan_offset(NX_NUMBER):
+            unit: NX_LENGTH
+            exists: optional
+            doc: |
+              The scan offset (in STM experiment) is the position of the tip in the scan area.
+              (e.g. -2.354637E-7 1.267476E-)
```

```suggestion
               In STM experiments, the scan offset is the position of the tip in the scan area (e.g. -2.354637E-7m 1.267476E-7).
```
Limit dimension if appropriate (see above).

In general, these docstring could be improved. What is the scan offset? The position of the tip at the scan start? The distance to the middle of the sample? It surely cannot be the position of the tip because that is scanned.

---

Doc is improved!